### PR TITLE
as2-32 added new test and method

### PR DESCRIPTION
**Pull request by SoniaMiach**

### CODE REVIEW

File: src/test/java/api/clients/ListTestRestClient.java [hunk 1]
```diff
@@ -61,7 +62,25 @@ public ListsDataResponse moveList(String listId, String boardId) {
...
+                });
+    }
+
     public static Map<String, String> constructDefaultListKeyValue() {
         return Map.of("name", "Test list " + RandomStringUtils.randomAlphanumeric(2));
     }
+
+    public static Map<String, String> moveAllCardsToListKeyValue(String boardId, String listId){
+        return Map.of("idBoard" , boardId , "idList" , listId );
+    }
```
Comment (Goldbe54) [inline src/test/java/api/clients/ListTestRestClient.java]: Rename to constructMoveAllCardsToListKeyValue

File: src/test/java/api/post/MoveAllCardsInList.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    @BeforeMethod
+    @Step("Fulfillment of the prerequisites for the test")
+    public void preconditions() {
+        boardId = boardRestTestClient.createNewBoard(constructDefaultBoardKeyValue()).getId();
+        listId = listTestRestClient.createList(constructDefaultListKeyValue(), boardId).getId();
+        cardId = cardTestRestClient.createCard(constructDefaultCardKeyValue(), listId).getId();
+        secondCardId = cardTestRestClient.createCard(constructDefaultCardKeyValue(), listId).getId();
+        secondBoardId = boardRestTestClient.createNewBoard(constructDefaultBoardKeyValue()).getId();
+        secondListId = listTestRestClient.createList(constructDefaultListKeyValue(), secondBoardId).getId();
+
```
Comment (Goldbe54) [inline src/test/java/api/post/MoveAllCardsInList.java]: delete empty line

File: src/test/java/api/post/MoveAllCardsInList.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    @Test(description = "AS2-32")
+    @Description("Positive: Move all Cards in List")
+    public void moveCardsInList() {
+        List<CardDataResponse> response = listTestRestClient.moveAllCardsInList(moveAllCardsToListKeyValue(secondBoardId, secondListId), listId);
+
+        for (CardDataResponse card : response) {
+            Assert.assertEquals(card.getIdList(), secondListId, "Card was not moved to the target list");
```
Comment (Goldbe54) [inline src/test/java/api/post/MoveAllCardsInList.java]: try use:
Assert.assertEquals(response,get(0).getIdList(), secondListId, "Card was not moved to the target list");

Without:
 for (CardDataResponse card : response)

**Review by Goldbe54 — APPROVED** [review]

File: src/test/java/api/post/MoveAllCardsInList.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+    private String boardId;
+    private String listId;
+    private String cardId;
+    private String secondCardId;
+    private String secondBoardId;
+    private String secondListId;
+
+    @BeforeMethod
+    @Step("Fulfillment of the prerequisites for the test")
```
Comment (KhraschukZahar) [inline src/test/java/api/post/MoveAllCardsInList.java]: Better use @Step("Test preparation") or @Step("Preparing to the test")

**Review by KhraschukZahar — APPROVED** [review]

**Pull request merged**